.php_cs

```diff
@@ -1,44 +0,0 @@
-<?php
-
-$header = <<<EOF
```

Comment:
IMO We should keep this file. We are keep `SimpleThings\EntityAuditBundle` so header should be keep. We can add note about we are maintainer since xxx. 

WDYT? @sonata-project/contributors

Reply:
All these files will be autogenerated by dev-kit, so it's easier this way.

I prefer to add a custom note in the README instead.

Reply:
I did not think about it. We can do it too.